CHANGELOG.md

```diff
@@ -128,11 +128,53 @@ the APIs in these libraries are stable, and are ready for production use.
 
 - [Cloud Support API](/google/cloud/support/README.md)
 
+The following experimental libraries are now available:
+
+- [Compute Engine API](/google/cloud/compute/README.md)
+
+### [Bigtable](/google/cloud/bigtable/README.md)
+
+- docs(bigtable): async Table APIs are stable ([#11711](https://github.com/googleapis/google-cloud-cpp/pull/11711))
+- doc(bigtable): deprecate DataClient in doxygen ([#11550](https://github.com/googleapis/google-cloud-cpp/pull/11550))
+
+### [Cloud Asset](/google/cloud/asset/README.md)
+
+The library has been re-enabled on Windows. See
+[#11714](https://github.com/googleapis/google-cloud-cpp/issues/11714) for details.
+
+### [Spanner](/google/cloud/spanner/README.md)
+
+- docs(spanner): some documentation tweaks ([#11641](https://github.com/googleapis/google-cloud-cpp/pull/11641))
+- feat(spanner): tests and samples for drop-database protection ([#11637](https://github.com/googleapis/google-cloud-cpp/pull/11637))
+- feat(spanner): make `ResultSourceInterface` public ([#11636](https://github.com/googleapis/google-cloud-cpp/pull/11636))
+
+### [Storage](/google/cloud/storage/README.md)
+
+- docs(storage): add GCS+gRPC release notes ([#11760](https://github.com/googleapis/google-cloud-cpp/pull/11760))
```

Comment:
I think you are missing https://github.com/googleapis/google-cloud-cpp/pull/11754